triage-party/release-team/ingress.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: networking.k8s.io/v1beta1
+apiVersion: extensions/v1beta1
```

Comment:
why change away from networking.k8s.io?

Reply:
never mind, I saw why, thought I had deleted this comment... we should have an open issue about this